examples/00_quick_start/fbt_movielens.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
<p>it looks this cell hasn't been executed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Im trying to access ReviewNB but I don't see it...it seems to be a permissions thing

Reply:
this problem is resolved. Have access to reviewNB now.